pkg/auth/service.go

```diff
@@ -1273,9 +1281,13 @@ func (a *APIAuthService) ListUsers(ctx context.Context, params *model.Pagination
 	results := resp.JSON200.Results
 	users := make([]*model.User, len(results))
 	for i, r := range results {
+		username := r.Username
+		if username == "" {
+			username = swag.StringValue(r.Name)
```

Comment:
We would like to return the Name as Username even if it is empty, as we will make it required. Unless we plan on the next PR remove this requirement.

Reply:
The username is in the Auth service API definition - here is the lakeFS OpenAPI definition - no Name field.